### PR TITLE
Forward the same env variables as Builder does

### LINKED ISSUE #22: Forward the host env variables when running the app
I've noticed that [vorta](https://github.com/borgbase/vorta) builds fine with Builder, but fails with flatpak-vscode with the following error:

```bash
qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```

This is because flatpak-vscode executes:
```bash
flatpak build --with-appdir --allow=devel \
--bind-mount=/run/user/1000/doc=/run/user/1000/doc/by-app/com.borgbase.Vorta \
 --socket=fallback-x11 --share=ipc --socket=wayland --device=dri \
 --share=network --filesystem=host --own-name='org.kde.*' \
 --talk-name='org.freedesktop.DBus.*' --talk-name='org.freedesktop.Flatpak.*' \
 --talk-name=org.freedesktop.secrets --talk-name=org.kde.kwalletd5 \
 --socket=ssh-auth --talk-name=org.freedesktop.Notifications \
 --system-talk-name=org.freedesktop.NetworkManager \
 --talk-name='org.freedesktop.portal.*' --talk-name=org.a11y.Bus \
 /var/home/julian/Projekte/Others/vorta/.flatpak/repo vorta
```
while Builder executes :
```bash
flatpak-spawn --host flatpak build --with-appdir --allow=devel \
--bind-mount=/run/user/1000/doc=/run/user/1000/doc/by-app/com.borgbase.Vorta \
--socket=fallback-x11 --share=ipc --socket=wayland --device=dri \
 --share=network --filesystem=host --own-name='org.kde.*' \
 --talk-name='org.freedesktop.DBus.*' --talk-name='org.freedesktop.Flatpak.*' \
 --talk-name=org.freedesktop.secrets --talk-name=org.kde.kwalletd5 \
 --socket=ssh-auth --talk-name=org.freedesktop.Notifications \
--system-talk-name=org.freedesktop.NetworkManager \
 --talk-name='org.freedesktop.portal.*' --talk-name=org.a11y.Bus 
```
starting from here come the differences (retracted a few obviously Builder specific commands and emphasized the differences)
```
--env=COLORTERM=truecolor --env=DESKTOP_SESSION=gnome \
--env=LANG=de_AT.UTF-8 --env=WAYLAND_DISPLAY=wayland-0 \
--env=XDG_CURRENT_DESKTOP=GNOME \
--env=XDG_SESSION_DESKTOP=gnome \
--env=XDG_SESSION_TYPE=wayland \
--env=AT_SPI_BUS_ADDRESS=unix:path=/run/user/1000/at-spi-bus \
--env=TERM=xterm-256color \
 /var/home/julian/Projekte/Others/vorta/.flatpak/repo vorta
```

Note: the relevant part in that specific case is ```--env=XDG_SESSION_TYPE=wayland```

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Fixes #22

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -140,3 +140,27 @@ export const exists = async (path: string): Promise<boolean> => {
     return false
   }
 }
+
+export const getEnvVars = (): Map<string, string> => {
```
Comment (bilelmoussaoui) [inline src/utils.ts:144]:
```suggestion
export const getHostEnv = (): Map<string, string> => {
```
maybe?

File: src/utils.ts [hunk 2]
```diff
@@ -140,3 +140,27 @@ export const exists = async (path: string): Promise<boolean> => {
...
+  const envVarKeys: string[] = [
```
Comment (bilelmoussaoui) [inline src/utils.ts:145]: forwardedEnvKeys ?

File: src/utils.ts (hunk 1 above)
Comment (Hofer-Julian) [reply src/utils.ts:144]: Sure

File: src/utils.ts (hunk 2 above)
Comment (Hofer-Julian) [reply src/utils.ts:145]: Fine as well :)

**Pull request merged**